src/docs/sdk/features.mdx

```diff
@@ -154,3 +154,17 @@ This is mostly useful on mobile clients where connectivity is often not availabl
 ## HTTP Proxy
 
 Ability to use an HTTP proxy. Often easy to implement using the existing HTTP client.  This should be picked up from the system config if possible or explicit config in the client options.
+
+## HTTP Client Integrations
+
+Add a breadcrumb for each outgoing HTTP request after the request finishes:
+
+- type: `http`
+- category: `http`
+- data: `url`, `method` (uppercase HTTP method), optionally `requestBodySize` and `responseBodySize` (in bytes)
+
+If Performance Monitoring is both supported by the SDK and enabled in the client application when the transaction is active a new `Span` must be created around the HTTP request.
```

Comment:
describe span `op` and `desc`
eg `op=http.client` and `desc=$method $url`

also the OkHttp integration now has a `BeforeSpanCallback` which allows customizing the `span`

Reply:
> describe span op and desc
> eg op=http.client and desc=$method $url

You mean just adding these examples? What description do you have in mind?

> also the OkHttp integration now has a BeforeSpanCallback which allows customizing the span

What's the suggestion? To add a note that integrations can take a callback to modify the span directly in a way that the HTTP Context is provided to the callback?

Reply:
the PR looks good to me and got 👍 from @kamilogorek and other folks tagged had 1 month to react.
Happy to merge as-is or feel free to add the suggestions, commit them and merge.

🚀 🚀

Reply:
my suggestion was just to explain that the `op` should be `http.client` and the description of the span should follow the convention `$method $url` eg `GET https://sentry.io`, so we don't diverge on that.

the `BeforeSpanCallback` can be added later since we're waiting feedback about it yet

Reply:
```suggestion
If Performance Monitoring is both supported by the SDK and enabled in the client application when the transaction is active a new `Span` must be created around the HTTP request; The `Span#operation` should be `http.client` and the `Span#description` should follow the convention `$METHOD $url` eg `GET https://sentry.io`.
```